packages/integrations/image/components/Image.astro

```diff
@@ -2,29 +2,11 @@
 // @ts-ignore
 import { getImage } from '../dist/index.js';
 import { warnForMissingAlt } from './index.js';
-import type { ImgHTMLAttributes } from './index.js';
-import type { ImageMetadata, TransformOptions, OutputFormat } from '../dist/index.js';
+import type { ImageComponentLocalImageProps, ImageComponentRemoteImageProps } from './index.js';
 
-interface LocalImageProps
-	extends Omit<TransformOptions, 'src'>,
```

Comment:
For some reason, when generics are used directly inside the astro component, its props become Record<string, any>. When extracted to another file, it works as expected

Reply:
Our language server doesn't support generics at the moment, but it will soon. Your workaround seems valid, though!

cc @Princesseuh for a TS review 😄